backend/services/mail_send.py

```
@@ -0,0 +1,76 @@
+
+    def send_bulk_mail(self, title: str, content: dict[str, Any], recipient: List[str], template_name: str, message="", sender=None):
+        """
+            method to send bulk mails
+        """
+        try:
+            if sender is None:
+                sender = self.sender
+            content = self._create_message(template_name, content)
+            logger.info(f"Sending mail to {recipient} {content}")
+            send_mass_mail(title, message,from_email=sender, recipient_list=recipient, html_message=content, fail_silently=False)
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)